tasks/TaskTypes.py

```diff
@@ -22,5 +22,4 @@ class TaskType(enum.Enum):
     )  # This is a specific type of text classification with unique properties
     PARAPHRASE_DETECTION = (13,)
     TEXTUAL_ENTAILMENT = (14,)
-    QUALITY_ESTIMATION = (15,)
-
+    QUALITY_ESTIMATION = (15,)
```

Comment:
Also this has been added. You might want to add it below in the transformation's `tasks` variable .

Reply:
Just added the task (along with the keywords and evaluation on the transformation's readme)

About the adverbs of indefinite frequency, I feel it would make more sense as its own transformation. In that case you can switch them for an equivalent but opposing adverb instead of just removing them (e.g. "never"->"alway" or "normaly"->"seldom"). The rules to do that would also be slightly different (simpler). Are new transformations still being accepted?